lib/registries/addon/drafts/draft/form/route.ts

```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default class DraftsFormRoute extends Route.extend({}) {
+    model() {
+        return this.modelFor('drafts');
```

Comment:
Instead of a model hook that will definitely need to be rewritten, it might be nice to add page tracking to each new route, since that's a consistent need and is easy to forget. Something like:
```ts
export default class ... extends Route {
    @service analytics!: Analytics;

    @action
    didTransition() {
        this.analytics.trackPage();
    }
}
```